python/tvm/relax/frontend/onnx_frontend.py

```diff
@@ -329,99 +333,99 @@ class BiasGelu(OnnxOpConverter):
     """
 
     @classmethod
-    def _impl_v1(cls, bb, inputs, attr):
-        inp = relax.op.add(inputs[0], inputs[1])
-        return relax.op.nn.gelu(inp)
+    def _impl_v1(cls, bb, inputs, attr, span):
+        inp = relax.op.add(inputs[0], inputs[1], span=span)
+        return relax.op.nn.gelu(inp, span=span)
 
 
 class Where(OnnxOpConverter):
     """Convert an onnx Where node into an equivalent Relax expression."""
 
     @classmethod
-    def _impl_v16(cls, bb, inputs, attr):
-        return relax.op.where(inputs[0], inputs[1], inputs[2])
+    def _impl_v16(cls, bb, inputs, attr, span):
+        return relax.op.where(inputs[0], inputs[1], inputs[2], span=span)
```

Comment:
I wonder if a potentially cleaner way to do this would be with a span context. Then we could do something like this:
```
with span_context("op_name"):
    self.op_converter(op)
```

The op converters would then not need to acknowledge the span, instead in the python definition of the op we could have:
```
if span_context.current != None:
    span = span_context.span
ffi.make(op, span=span)
```

What do you think about this approach vs the current one?

Reply:
I agree that this is a cleaner solution. I've added a SpanContext in tvm/relax/utils.py. I am leaving all the operators to take a span as an optional argument -- in case we want at any point to use a different span than the one in the context.